### ISSUE TITLE
Add a 'Waiting..' modal when approve token/sign transaction in the expert mode

### ISSUE THREAD

**Issue by elena-zh** [gnosis/cowswap#1137]
It would be nice to add a  a 'Waiting..' modal when we send a request to a connected wallet for a token approval like we do for signing swap transaction. 

Also, it would be nice to have such modal when signing transaction in the Expert mode. 

![nice to show.jpg](https://images.zenhubusercontent.com/5f55ec970a2e890ec10f7778/8e558ed9-c55c-4fec-b5f6-3d61edf1f8ef)
![nice to show with result.jpg](https://images.zenhubusercontent.com/5f55ec970a2e890ec10f7778/2018b52d-c5f5-40e2-a3c3-4fcc8195e53f)

Once a transaction is approved/rejected, show a confirmation/error modal
![image.png](https://images.zenhubusercontent.com/5f55ec970a2e890ec10f7778/d959c018-4caa-44de-8e68-99258729f029)
![image.png](https://images.zenhubusercontent.com/5f55ec970a2e890ec10f7778/d4a1f208-1df6-40b1-bee6-44dc0e163d63)

**Comment by elena-zh** [comment]
Might be related to #606 ?
